data/map planets.txt

```diff
@@ -2028,6 +2043,13 @@ planet Janiculum
 	government Uninhabited
 	security 0
 
+planet Jupiter
+	attributes "gas giant" "requires: gaslining" uninhabited
+	landscape land/nasa12
+	description `Once called the King of the Solar System, Jupiter is just as impressive even considering the variety of gas giants in the galaxy. Its famous Great Red Spot is still well-known by most of humanity, and you make sure to steer clear of it while taking in the rich yellows and browns of the planet's atmosphere.`
```

Comment:
Yeah, I'm not sure about that either. I just thought I should mention it.

Reply:
The Great Red Spot is visible in Jupiter's planet sprite, so in that sense it's canon to ES that it's still around, whether or not that's realistic.

Reply:
Jupiter may have grown a new storm which is similarly red, who knows?

Reply:
I actually think it would be funny if it would be gone.
Showing that the contributors of the game actually take real life data into account(which we do not)